src/ios/DateTimePicker.m

```diff
@@ -96,11 +96,12 @@ - (void)initPickerView:(UIView*)theWebView {
 
 - (void)configureDatePicker:(NSMutableDictionary *)optionsOrNil datePicker:(UIDatePicker *)datePicker {
     
-    // Prefer wheels style on iOS 14.
-    if (@available(iOS 13.4, *)) {
-        datePicker.preferredDatePickerStyle = UIDatePickerStyleWheels;
-    }
-    
+    // Prefer wheels style on iOS 14.
+    if (@available(iOS 13.4, *)) {
+        datePicker.preferredDatePickerStyle = UIDatePickerStyleWheels;
+        [datePicker addTarget:self action:@selector(handleDatePickerTap:) forControlEvents:UIControlEventEditingDidBegin];
```

Comment:
Should this rather be moved to `initPickerView`? This method is called on each 'show', and thus registers the delegate each time, which might be leaky, can you confirm?